docs/src/pages/style/typography/TypographyFontFamily.js

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { createMuiTheme, CssBaseline } from '@material-ui/core';
+import { ThemeProvider } from '@material-ui/styles';
+import RalewayLightWoff2 from '@material-ui/docs/src/pages/style/typography/fonts/Raleway-Light.woff2';
```

Comment:
> I read the results of the Material UI survey this weekend and saw that a custom typography demo was asked for.

We already document it. I think that it's more about people who are struggling to discover the content than anything else. Your example is about self-hosting a font, it's a different concern. 

![Capture d’écran 2019-03-18 à 11 08 24](https://user-images.githubusercontent.com/3165635/54522723-2c4d3b80-496e-11e9-923c-7ca94862ff40.png)

> I could just switch this to use Google's CDN as I mention in the code comments and then write a detailed comment in the block at the top on how you'd self-host. I'm open to w/e is easier or better for everyone.

1. I think that it would be great to move the `// Use the system font instead of the default Roboto font.` comment from the code to the markdown, so it can be searchable with Algolia.
2. I'm happy to add a documentation section about how to declare a custom font with JSS. It's not the first time I see this concern. But I don't think that we should add a demo for it. I would rather keep it simple cc @mbrookes.

Reply:
After re-reading the blog post, I clearly had non-traditional approaches to changing the font on my mind when reading the survey. `Non-material font example` definitely fits the existing demo. What I think it lacks is the bridge to a font other than system or Roboto using overrides and JSS. I certainly had that question when I started using MUI and it took a while for me to learn enough of both to make it happen. I think it would be beneficial to include something small to improve Material-UI's 'turn key' approach.

Reply:
I just removed the demos and altered the theme customization docs to simplify things a bit more.